### ISSUE TITLE
Landing page | Table alignment

### ISSUE THREAD

**Issue by gnakm** [department-of-veterans-affairs/caseflow#2820]
## Description
Right now the text and button aligned in a row are not aligned and it's hard to follow when reading across. 

## AC
- [ ] Verify that all text and button are aligned at the top. 

## Mockup
![image](https://user-images.githubusercontent.com/13420618/28885956-b4397a7a-7785-11e7-8c5e-74bb3df924a6.png)

**Comment by mmlumba** [comment]
@gnakm What's the space between the red line and the divider underneath the column names? 30px?

**Comment by gnakm** [comment]
Marian and I discussed in person that we should use default style guide spacing.

**Comment by astewarttistatech** [comment]
**PASSED**

Env: UAT
Browser: FF and Chrome 

Steps to Validate: 
1. Log into Reader with a Judge or Attorney's css_id
2. Navigate to the Welcome Gate
3. Verify that the all of the table is top aligned 

Screenshots: 
![readertablealign1](https://user-images.githubusercontent.com/23038505/29222401-519dd46c-7e90-11e7-9093-c4d731f4fb45.png)
